src/controllers/managing.officer.controller.ts

```diff
@@ -17,7 +17,11 @@ export const get = (req: Request, res: Response) => {
 };
 
 export const getById = (req: Request, res: Response, next: NextFunction) => {
-  getManagingOfficerById(req, res, next, config.BENEFICIAL_OWNER_TYPE_URL, config.MANAGING_OFFICER_PAGE);
+  let backLinkUrl: string = config.BENEFICIAL_OWNER_TYPE_URL;
```

Comment:
Could you code this consistently with the other two changes and use a constant for `backLinkUrl` and assign value directly using the `?` operator?

Reply:
done